README.md

```diff
@@ -20,68 +20,101 @@
 
-Users will be prompted to upload a file containing an image of their document. On handheld devices they can also use the native camera to take a photo of their document.
+- Carefully designed UI to guide your customers through the entire photo and video capture process
+- Modular design to help you seamlessly integrate the photo and video capture process into your application flow
+- Advanced image quality detection technology to ensure the quality of the captured images meets the requirement of the Onfido identity verification process, guaranteeing the best success rate
+- Direct image upload to the Onfido service, to simplify integration
 
-Face step allows users to use their device cameras to capture their face using photos or videos.
+⚠️ Note: the SDK is only responsible for capturing photos and videos. You still need to access the [Onfido API](https://documentation.onfido.com/) to manage applicants and perform checks.
 
 ![Various views from the SDK](demo/screenshots.jpg)
 
-## Getting started
+## Getting started 
+
+### 1. Obtain an API token
 
-### 1. Obtaining an API token
+In order to start integrating, you'll need an [API token](https://documentation.onfido.com/#api-tokens). 
 
-In order to start integration, you will need the **API token**. You can use our [sandbox](https://documentation.onfido.com/#sandbox-testing) environment to test your integration, and you will find the sandbox token inside your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens).
+You can use our [sandbox](https://documentation.onfido.com/#sandbox-testing) environment to test your integration. To use the sandbox, you'll need to generate a sandbox API token in your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens). 
 
 #### 1.1 Regions
 
 Onfido offers region-specific environments. Refer to the [Regions](https://documentation.onfido.com/#regions) section in the API documentation for token format and API base URL information.
 
-### 2. Creating an applicant
+### 2. Create an applicant
 
-With your API token, you should create an applicant by making a request to the [create applicant endpoint](https://documentation.onfido.com/#create-applicant) from your server:
+To create an applicant from your backend server, make request to the ['create applicant' endpoint](https://documentation.onfido.com/#create-applicant), using a valid API token.
+
+⚠️ Note: Different report types have different minimum requirements for applicant data. For a Document or Facial Similarity report, the minimum applicant details required are `first_name` and `last_name`.
 
 ```shell
 $ curl https://api.onfido.com/v3/applicants \
-  -H 'Authorization: Token token=YOUR_API_TOKEN' \
+  -H 'Authorization: Token token=<YOUR_API_TOKEN>' \
   -d 'first_name=John' \
   -d 'last_name=Smith'
 ```
 
-Note: If you are currently using API `v2` please refer to [this migration guide](https://developers.onfido.com/guide/api-v2-to-v3-migration-guide) for more information.
```

Comment:
Are you proposing to remove this because newer versions won't be compatible with v3 or for another reason / or is it a minor oversight?

Reply:
My thought was to include it at the beginning as a single statement/disclaimer, as at the moment it is included at various points but a bit buried in the content. Plus, it will be more manageable if compatibility does change in the future. But it seems I never went back and added it in so thanks for highlighting!